apim/3.x/README.adoc

```diff
@@ -150,18 +186,37 @@ define the following configuration options:
 |`+mongo.socketKeepAlive+` |Enable keep alive for socket |`+false+`
 |===
 
-==== Mongo ReplicaSet
+==== Mongo Replica Set
 
 [cols=",,",options="header",]
 |===
 |Parameter |Description |Default
-|`+mongodb-replicaset.enabled+` |Enable deployment of Mongo replicaset
+|`+mongodb-replicaset.enabled+` |Enable deployment of Mongo replica set
 |`+false+`
 |===
 
 See
 https://github.com/helm/charts/tree/master/stable/mongodb-replicaset[MongoDB
-replicaset] for detailed documentation on helm chart.
+replica set] for detailed documentation on Helm chart.
+
+
+==== PostgreSQL Database (via JDBC Connection) 
+To install a new PostgresSQL database, use the command below and update the `username`, `password`, and `databasename` parameters: 
+----
+helm install --set postgresqlUsername=postgres --set postgresqlPassword=P@ssw0rd 
+--set postgresqlDatabase=graviteeapim postgres-apim bitnami/postgresql
+----
+
+NOTE: When you install APIM, it automatically uses the default value for the host and runs on apim.example.com. To change this default value to a specific host name for your APIM installation, you must modify the `gateway.ingress.hosts` parameter (listed in the link:/apim/3.x/apim_installguide_kubernetes.html#gravitee_gateway[Gravitee Gateway table] below) in the `values.yml` configuration file. 
```

Comment:
If users want to use a custom hostname, they have to change multiple values in their configuration file, not only one : there is a discinct hostname configuration for each app (gateway/api/console/portal).
For example, you can search in the documentation all the values which have `[apim.example.com]` as default.